monasca-agent-base/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
for what its worth we don't need a build.sh in here... I really need to go through and delete all of these